posts/_markdown/quantumalgorithmsorg.md

```diff
@@ -0,0 +1,41 @@
+
+Quantumalgorithms.org has constantly been growing. The project is currently supported by the unitary.fund and the Centre For Quantum Technologies (the CQT of Singapore). These funds are dedicated to compensating collaborators that help us write more high-quality content and enrich the book's quality. At the moment, two collaborators are working on some brand-new, thrilling chapters that should be available by the end of the year!
+
+The book has already been used as teaching material for two different courses at Politecnico di Milano. We hope [quantumalgorithms.org](https://quantumalgorithms.org) can be used fruitfully in many other courses, summer schools, workshops, etc., in the future. Do you plan to teach quantum computing and quantum algorithms and need some lecture notes to support your teaching? Let us know! We would be extremely happy to cite your course on our website, share our teaching experience based on these lecture notes, and hand out additional material not yet transcribed on the website.
+
+This is just the beginning of our journey. In the future, the book will cover more content: new quantum algorithms and more appendices to cover advanced theoretical computer science topics. Interested MSc and Ph.D. students are encouraged to engage with the core team of the open-source project (contacts below!). We need your help in building this giant book! There are many big and small issues on GitHub that are waiting for you! New chapters are waiting to be written, and many sections could be expanded or ameliorated.
+ 
+We are eagerly waiting for different contributors: Typo Hunters, Friendly Reviewers, Consistency Fanatics, and Happy Writers. To name some tasks, we are searching for contributors to improve the section on error propagation and write more introductory material in chapters 2 and 3. Another much-appreciated contribution would be to give us feedback on the book's content: are there parts that are too hard to understand? Can a section be made more easily accessible? Are we missing some algorithm or paper to cite? We also have a list of exercises that are waiting to be added to the book. Are you willing to go the extra mile, write a whole new chapter, and get paid for your contribution? Contact us, and let's have a chat! We are waiting for you!
```

Comment:
I think this comes off not how you intend, maybe something like "As the book grows, we are encouraging all kinds of contributors:...

Also, I am not quite sure what a happy writer is, :)

Making the tasks a list may make it easier to brows/catch people's attention.

Reply:
Done! :) Thank you again!